lib/nc_time_axis/__init__.py

```diff
@@ -210,4 +256,9 @@ def convert(cls, value, unit, axis):
         if not hasattr(first_value, 'calendar'):
             raise ValueError('A "calendar" attribute must be attached to '
                              'netcdftime object to understand them properly.')
-        return netcdftime.date2num(value, cls.standard_unit, calendar=first_value.calendar)
+        return netcdftime.date2num(value, cls.standard_unit,
+                                   calendar=first_value.calendar)
+
+
+if netcdftime.datetime not in munits.registry:
+    munits.registry[netcdftime.datetime] = NetCDFTimeConverter()
```

Comment:
To be fair, this was part of the original codebase. A comment wouldn't go amiss though...